src/main/java/de/slimecloud/slimeball/features/staff/TeamMeeting.java

```diff
@@ -122,7 +122,7 @@ public synchronized void onButtonInteraction(@NotNull ButtonInteractionEvent eve
 					if(config.getEvent() != 0) event.getGuild().retrieveScheduledEventById(config.getEvent()).flatMap(ScheduledEvent::delete).queue();
 
 					//This will send a new message
-					config.createNewMeeting(current.getTimestamp().toLocalDateTime().plus(Duration.ofDays(14)).toInstant(Main.timezone));
+					config.createNewMeeting(current.getTimestamp().toLocalDateTime().plus(Duration.ofDays(14)).atZone(Main.timezone).toInstant());
```

Comment:
it is every time you end a meeting 2h behind
so if you add this you can fix it until Nov after Nov he will always add 1h

```suggestion
					config.createNewMeeting(current.getTimestamp().toLocalDateTime().plus(Duration.ofDays(14)).plus(Duration.ofHours(2)).atZone(Main.timezone).toInstant());
```